jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -115,6 +115,7 @@ function main() {
       -p /usr/share/jenkins/ref/plugins
       --runHome "${_JENKINS_HOME}"
       --no-sandbox
+      --write-log-init-timeout 120
```

Comment:
I think that should be 20s.

Similar to https://github.com/SAP/stewardci-jenkinsfilerunner-image/commit/121b0731c78e5624f610422bc124f613bb496961